nfc/NfcCx/Cx/NfcCxSE.h

```diff
@@ -32,6 +32,13 @@ EXTERN_C __declspec(selectany) const GUID DH_SECUREELEMENT_ID =
 
 #define NFCCX_MAX_HCE_PACKET_QUEUE_LENGTH (5)
 
+typedef enum _CARD_EMULATION_MODE_INTERNAL {
+	CardEmulationOff = 0,
+	CardEmulationOnPowerIndependent = 1,
+	CardEmulationOnPowerDependent = 2,
+	EmbeddedSeApduMode = 3
+} CARD_EMULATION_MODE_INTERNAL;
+
```

Comment:
this is the new macro which needs to be added to the Card emulation mode for eSE  Since the eSE code is not pushed in to mainline, an internal enum is created. The internal enum will be deleted once the eSE mode is available in the mainline